matchms/similarity/ModifiedCosine.py

```diff
@@ -93,11 +94,20 @@ def get_matching_pairs():
             nonzero_pairs = collect_peak_pairs(spec1, spec2, self.tolerance, shift=mass_shift,
                                                mz_power=self.mz_power,
                                                intensity_power=self.intensity_power)
-            unsorted_matching_pairs = zero_pairs + nonzero_pairs
-            return sorted(unsorted_matching_pairs, key=lambda x: x[2], reverse=True)
+
+            if zero_pairs is None:
```

Comment:
There is no test which resulted in zero_pairs to be filled.

Reply:
Weird.  that case will occur in the tests, not sure why sonarcloud does not recognize it though.

Reply:
In noticed them when I run `pytest --cov --cov-report html` then there are yellow lines in htmlcov/*html. Maybe you can only run the test you expect to cover that line to see what the coverage looks like.

On the other hand having 99% coverage is very good already, we can also let it go.

Reply:
I would indeed leave it at that for now.